web/pgadmin/dashboard/static/js/Dashboard.jsx

```diff
@@ -833,6 +839,26 @@ export default function Dashboard({
     );
   };
 
+  const CustomActiveOnlyHeaderLabel =
```

Comment:
No need to create variable. Pass the object directly

Reply:
Yeah I tried that on my side, and I could not get that to work reliably for me, which is why I ended up creating it as a variable so that I could get getText() to work correctly. I will review these changes, incorporate them into my branch, and then update my pull request. Thank you for your help with this.